README.markdown

```diff
@@ -115,6 +141,17 @@ Then, create the schema:
 $ psql faces < schema.sql
 ```
 
+Add your database connection URL to the `.env` file:
+`DATABASE_URL=postgresql://localhost/`
+
+**Note**: the `DATABASE_URL` can be any
+[libpq connection string](https://www.postgresql.org/docs/current/static/libpq-connect.html#LIBPQ-CONNSTRING).
+Alternate database URLs you might try are
+`postgres://localhost/faces`
+or
+`postgres:///faces`,
+to connect over TCP/IP to the database named `faces`.
```

Comment:
Sorry, now that I reread this, I was very unclear about what my concern was here!

Right now, this sentence says:

>  Alternate database URLs you might try are `postgres://localhost/faces` or `postgres:///faces`, to connect over TCP/IP to the database named `faces`.

The former connects via TCP/IP, but the latter connects via a Unix domain socket, so the explanation at the end is wrong.

Perhaps it could read:

> An alternate database URL you might try is: `postgres://localhost/faces`  to connect over TCP/IP to the database named `faces`.

Reply:
Ahh, sorry for the misunderstanding! Corrected with your suggestion 😄